### PR TITLE
Fix version and changelog

### LINKED ISSUE #752: Update sdk/template/azure-template/CHANGELOG.md so that the versions are listed in the right order.
See https://github.com/Azure/azure-sdk-for-cpp/pull/725#discussion_r503439709

Update [sdk/template/azure-template/CHANGELOG.md](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/template/azure-template/CHANGELOG.md) so that the versions are listed in the right order.

Right now they are not in the right order:
image

### PR DESCRIPTION
**Pull request by RickWinter**
Closes #752

### CODE REVIEW

File: sdk/template/azure-template/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,7 @@
 # Release History
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.8 (Unreleased)
```
Comment (ahsonkhan) [inline sdk/template/azure-template/CHANGELOG.md:2]: Wait for https://github.com/Azure/azure-sdk-for-cpp/pull/762 to go in, before making this version update change on top.

Comment (RickWinter) [reply sdk/template/azure-template/CHANGELOG.md:2]: 762 isn't correct.

Comment (ahsonkhan) [reply sdk/template/azure-template/CHANGELOG.md:2]: It is now.

**Comment by RickWinter** [comment]
@ahsonkhan @vhvb1989 @antkmsft please approve

**Review by ahsonkhan — APPROVED** [review]

**Review by weshaggard — APPROVED** [review]

**Pull request merged**